app/views/users/require_2sv.html.erb

```diff
@@ -1,16 +1,23 @@
-<% content_for :title, "Create new user" %>
+<% content_for :title, "2-step verification settings for new user" %>
 
-<h1>2-step verification settings for new user</h1>
-
-<%= form_for @user, :html => {:class => 'well'} do |f| %>
+<%= form_for @user do |f| %>
   <%= f.hidden_field :skip_update_user_permissions, value: "true" %>
-  <p class="checkbox">
-    <%= f.label :require_2sv do %>
-      <%= f.check_box :require_2sv %> Mandate 2-step verification for this user <%= "(this will remove their exemption)" if @user.exempt_from_2sv? %>
-    <% end %>
-    <br/>
-    User will be prompted to set up 2-step verification again the next time they sign in.
-  </p>
+  <%= render "govuk_publishing_components/components/hint", {
+    text: "User will be prompted to set up 2-step verification again the next time they sign in."
```

Comment:
Minor: I know you haven't changed this, but I'm a bit confused by the word "again" in this sentence:

> User will be prompted to set up 2-step verification again the next time they sign in.

Surely since this is in the user creation workflow, the user won't have ever set up their 2-step verification, so how can they be prompted to do it "again"...? Maybe I'm missing something!

Reply:
Yes, that is odd - I found this flow a little hard to understand in general and to be honest, adding this page to the design system while having the first "step" in the create user journey in the old design doesn't really help. I think it'd be worth seeing if we can improve this whole flow and the copy at some point.